### PR TITLE
Metalkube host tabs

### PR DESCRIPTION
**Pull request by honza**
Yaml, NICs, and disks

Depends on #216

### CODE REVIEW

File: frontend/public/components/resource-pages.ts [hunk 1]
```diff
@@ -117,7 +117,8 @@ export const resourceDetailPages = ImmutableMap<GroupVersionKind | string, () =>
   .set(referenceForModel(SubscriptionModel), () => import('./operator-lifecycle-manager/subscription' /* webpackChunkName: "subscription" */).then(m => m.SubscriptionDetailsPage))
   .set(referenceForModel(InstallPlanModel), () => import('./operator-lifecycle-manager/install-plan' /* webpackChunkName: "install-plan" */).then(m => m.InstallPlanDetailsPage))
   .set(referenceForModel(ClusterOperatorModel), () => import('./cluster-settings/cluster-operator' /* webpackChunkName: "cluster-operator" */).then(m => m.ClusterOperatorDetailsPage))
-  .set(referenceForModel(OAuthModel), () => import('./cluster-settings/oauth' /* webpackChunkName: "oauth" */).then(m => m.OAuthDetailsPage));
+  .set(referenceForModel(OAuthModel), () => import('./cluster-settings/oauth' /* webpackChunkName: "oauth" */).then(m => m.OAuthDetailsPage))
+  .set(referenceForModel(BaremetalHostModel), () => import('../metalkube/components/host' /* webpackChunkName: "baremetal-hosts" */).then(m => m.BaremetalHostsDetailPage));
```
Comment (mareklibra) [inline frontend/public/components/resource-pages.ts]: can you please move this change up, closer to the `VirtualMachineModel`? To simplify merge conflict resolution for openshift/console ...

File: frontend/public/metalkube/components/host/host-detail.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import { Row, Col } from 'patternfly-react';
+import { getResource } from 'kubevirt-web-ui-components';
+
+import { BaremetalHostModel } from '../../models/host';
+import { navFactory } from '../utils/okdutils';
+import { WithResources } from '../../../kubevirt/components/utils/withResources';
+import { DetailsPage, List, ListHeader, ColHead, ResourceRow } from '../factory/okdfactory';
+
+
+// TODO(honza): the BaremetalHostModel component should go into web-components once it's stable
```
Comment (mareklibra) [inline frontend/public/metalkube/components/host/host-detail.tsx]: please remove

Right, we plan to share the model definitions. Anyway, they will still be imported from a local .js file, i.e. via reexporting.

File: frontend/public/components/resource-pages.ts (hunk 1 above)
Comment (honza) [reply frontend/public/components/resource-pages.ts]: Sure thing, done.

File: frontend/public/metalkube/components/host/host-detail.tsx (hunk 2 above)
Comment (honza) [reply frontend/public/metalkube/components/host/host-detail.tsx]: Sorry, "model" was the wrong word. I meant the detail component just below. Comment removed none the less.

**Comment by honza** [comment]
Rebased on master

**Comment by jelkosz** [comment]
The dependent patch merged, removing blocked label

**Pull request merged**